ultimatejobweb/models.py

```diff
@@ -13,5 +13,13 @@ class Job(models.Model):
     job_title = models.CharField(max_length=128)
     description_url = models.URLField()
 
+    def push_data_api(company_name_api, title_list, url_list):
```

Comment:
Should this function be part of the Job class?
If so, it should be a class method and have the cls parameter first.
But it doesn't look to me like it belongs here

Reply:
@simzacks where do you think it should be ? 
It is a method to update new jobs

Reply:
> Should this function be part of the Job class?
> If so, it should be a class method and have the cls parameter first.
> But it doesn't look to me like it belongs here
@simzacks where do you think it should be ?
It is a method to update new jobs

Reply:
@BarBenNaim  please refer this sample PR
https://github.com/beyond-io/beyond-tutorial/pull/7/files#diff-d674dd14acda98e0ff388c113622f5224b22cde68f320bf7e6d39ce166214c09R11

Reply:
@Yarboa @simzacks 
I don't understand why I need class?